### PR TITLE
Language-Sensitive Representation of Numbers

### PR DESCRIPTION
**Pull request by DanielMwendwa**
Updated unique texts, Unique Texts with a label, Wrong Scheme messages, Not Coded messages columns on the coding progress table; Africas Talking Units Balance; Message metrics & Operator metrics sections in the traffic page.

### CODE REVIEW

File: public/js/controllers/traffic_metrics_controller.js [hunk 1]
```diff
@@ -34,8 +34,8 @@ export class TrafficMetricsController {
         let html = "<tr>%operator_metrics%</tr>";
         operators.forEach((operator) => {
             let tableData = `<td>${operator}</td>
-                <td>${d3.sum(data, (d) => d[`${operator}_received`])}</td>
-                <td>${d3.sum(data, (d) => d[`${operator}_sent`])}</td>`;
+                <td>${d3.sum(data, (d) => d[`${operator}_received`]).toLocaleString()}</td>
+                <td>${d3.sum(data, (d) => d[`${operator}_sent`]).toLocaleString()}</td>`;
```
Comment (as2388) [inline public/js/controllers/traffic_metrics_controller.js:38]: Thanks, these look better

File: public/js/controllers/traffic_metrics_controller.js [hunk 2]
```diff
@@ -49,7 +49,8 @@ export class TrafficMetricsController {
         let node = d3.select("#AT-units-bal");
         if (data.length) {
             let { balance } = data[0];
-            node.text(balance);
+            let [ currency, amount ] = balance.split(" ");
+            node.text(`${Number(amount).toLocaleString("en-KE", { style: "currency", currency })}`);
         } else {
             node.text("N/A");
```
Comment (as2388) [inline public/js/controllers/traffic_metrics_controller.js:58]: Drive-by comment for another PR: can this be "N/A" only if AfricasTalking isn't present in the project configuration? If it is, and there is no data, show an error here instead. This also means if we remove africastalking from the configuration this should switch back to "N/A"

File: public/js/controllers/traffic_metrics_controller.js [hunk 3]
```diff
@@ -49,7 +49,8 @@ export class TrafficMetricsController {
         let node = d3.select("#AT-units-bal");
         if (data.length) {
             let { balance } = data[0];
-            node.text(balance);
+            let [ currency, amount ] = balance.split(" ");
+            node.text(`${Number(amount).toLocaleString("en-KE", { style: "currency", currency })}`);
```
Comment (as2388) [inline public/js/controllers/traffic_metrics_controller.js]: Why is this en-KE when the others are the user's default? Is there a way to pass in the default (something like navigator.language)? I think it should be user's default everywhere, for consistency.

(If not then en-KE with an explanatory comment is fine - everyone in AVF should be comfortable with this locale)

**Review by as2388 — APPROVED** [review]

File: public/js/controllers/traffic_metrics_controller.js (hunk 2 above)
Comment (DanielMwendwa) [reply public/js/controllers/traffic_metrics_controller.js:58]: Noted

File: public/js/controllers/traffic_metrics_controller.js (hunk 3 above)
Comment (DanielMwendwa) [reply public/js/controllers/traffic_metrics_controller.js]: Thought it was better because it produces the 'Ksh' sign which we are used to instead of 'KES' code but it's not a big deal. Updated thanks.
